docs/notebooks/using_mclip.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "72867ba9-6a8c-4b14-acbf-487ea0a61836",
+   "metadata": {},
+   "source": [
+    "# Multilingual Text To Image search with MultilingualCLIP"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f576573b-a48f-4790-817d-e99f8bd28fd0",
+   "metadata": {},
+   "source": [
+    "Most text-image models are only able to provide embeddings for text in a single language, typically English. Multilingual CLIP models, however, are models that have been trained on multiple different languages. This allows the model the produce similar embeddings for the same sentence in multiple different languages.  \n",
+    "\n",
+    "This guide will show you how to finetune a multilingual CLIP model for a text to image retrieval in non-English languages.\n",
+    "\n",
+    "*Note, please consider switching to GPU/TPU Runtime for faster inference.*\n"
```

Comment:
Be more assertive. Check the runtime menu to me sure you are using a GPU./TPU instance, or this code will run very slowly.